### PR TITLE
Interactive dashboard implementation

### PR DESCRIPTION
**Pull request by gayanSandamal**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: docs/nodes/widgets/ui-chart.md [hunk 1]
```diff
@@ -96,8 +100,13 @@ The next most important properties to configure are the "Chart Type" and "X-Axis
 - **Chart Type**: Choose between a Line, Scatter, or Bar chart.
 - **X-Axis Type**: Choose between a "Timescale" (for time-based data), "Linear" (for numerical data), or "Categorical" (for non-numeric data). You'll notice that some x-axis types are only available for certain chart types.
 
+
```
Comment (joepavitt) [inline docs/nodes/widgets/ui-chart.md:103]: Screenshot 2024-07-26 at 11 14 14

Vertical alignment is off here

**Review by joepavitt — COMMENTED** [review]
Realising also the `#` navigation won't work because when you first arrive on the page, the groups haven't yet been loaded. 

So, we may want a "Table of Contents" or something like that in the first section so that users can jump to the relevant sections

File: docs/nodes/widgets/ui-chart.md (hunk 1 above)
Comment (gayanSandamal) [reply docs/nodes/widgets/ui-chart.md:103]: Removed these additional "Try Demo" button as per discussion

File: docs/components/TryDemo.vue [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+<style scoped lang="scss">
+.try-out-wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    .--btn {
+        background-color: var(--vp-c-brand-3);
+        ;
```
Comment (joepavitt) [inline docs/components/TryDemo.vue:31]:
```suggestion
```

File: docs/components/TryDemo.vue [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<template>
+    <div class="try-out-wrapper">
+        <slot></slot>
+        <a :href="`https://dashboard-demos.flowfuse.cloud/dashboard/${href}`" target="_blank" class="--btn">
+            {{ title }}
```
Comment (joepavitt) [inline docs/components/TryDemo.vue:5]: Would be good to have an External Link icon in htis button to indicate users will navigate away from the docs

**Review by joepavitt — CHANGES_REQUESTED** [review]

File: docs/components/TryDemo.vue (hunk 2 above)
Comment (gayanSandamal) [reply docs/components/TryDemo.vue]: Removed the additional semi-colon

File: docs/components/TryDemo.vue (hunk 3 above)
Comment (gayanSandamal) [reply docs/components/TryDemo.vue:5]: Screenshot 2024-07-26 at 22 14 52

Added the SVG icon from material icons as the Vuetify is only supports Dashboard at the moment

**Review by joepavitt — APPROVED** [review]

**Pull request merged**